.github/workflows/ci_ut_ext_blob_workflow.yml

```diff
@@ -0,0 +1,64 @@
+name: UT CI Run for Python Extension Blob
+
+on:
+    push:
+      branches: [ dev, master, main, release/* ]
```

Comment:
Same comments as the base extension workflow

Reply:
ditto